ReactWindows/ReactNative.Bridge/JavaScriptModuleBase.Generated1.cs

```diff
@@ -1,4 +1,11 @@
-﻿using System.Runtime.CompilerServices;
+﻿
+
+
+
+
+
+
+using System.Runtime.CompilerServices;
```

Comment:
Weird addition of whitespace all over the place in this file.  I know it's generated, but maybe it's a linebreak issue?